package.json

```diff
@@ -27,6 +27,7 @@
     "lodash": "^4.17.21",
     "node-expat": "^2.3.18",
     "node-fetch": "^2.6.7",
+    "orientjs": "^3.0.11",
```

Comment:
why was orientjs added? we shouldn't need it as a dependency since we go through the API for all the loaders

Reply:
Right. At first I thought adding the script I used to extract the data from the database, which is based on that dependency. After I decided not to include it in the PR. I Should have romeve taht depedency too.